libdevcore/JsonUtils.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <libdevcore/JsonUtils.h>
+#include <json_spirit/JsonSpiritHeaders.h>
+#include <set>
+#include <string>
+#include <ostream>
+
+void dev::validateFieldNames(json_spirit::mObject const& _obj, std::set<std::string> const& _allowedFields)
```

Comment:
could we do a function like this to validate test fillers for unknown fields?
or do you think that json scheme on tests repo is enough?

Reply:
I think the Json scheme on test repo is enough.